lib/hooks/docbase/helper.rb

```diff
@@ -0,0 +1,30 @@
+module Idobata::Hook
+  class Docbase < Base
+    module Helper
+      def link_to_post(post_title, post_url)
+        capture_haml do
+          haml_tag :a, post_title, href: post_url
+        end
+      end
+
+      def link_to_user(team_domain, user_id, user_name)
+        url = "https://#{team_domain}.docbase.io/users/#{user_id}"
+        capture_haml do
+          haml_tag :a, user_name, href: url
+        end
+      end
+
+      def link_to_joined_team_or_group(name, message)
+        url_first_index = message.index(/https:/)
+        url = message[url_first_index..-1]
```

Comment:
With `URI.extract` I think that the intention of the code will become clearer.

Reply:
Oops, `URI.extract` was deprecated. We could use `URI::RFC 2396_Parser#extract` instead.

Reply:
Thank you for review! Fix to use URI::RFC2396_Parser#extract.